docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/02_instrument_format/LINST_instrument.md

```diff
@@ -0,0 +1,156 @@
+- **Instrument Name**, **Date of Administration**, and **Age at administration** will be added automatically at the top of each instrument form.
+
+#### Data types in the Instrument Builder
+
+| Data Type | Description |
+| --- | --- |
+| **Header** | Specifies page title or section header. Text appears boldface at page centre. _**Note:** Instrument Name automatically becomes the header at the top of form._ |
+| **Label** | A formatting option to introduce a subset of questions. Labels appear as regular text in their own paragraphs.|
+| **Score Field** | Score fields are place holders for automated scoring values calculated by the scoring logic in the `.score` file. |
+| **Textbox** | Used for shorter miscellaneous text answers not covered by other options. |
+| **Textarea** | Used for longer inputs such as general comments, etc. |
+| **Dropdown** | Single select field where the list displayed is defined in the `Dropdown Options` section. _**Note:** A `not_answered` option is added automatically to all dropdowns._ |
+| **Multiselect** | A select box allowing multiple options to be chosen. |
+| **Date** | Used for creating a date field such as Date of Birth. _**Note:** Date of Administration is automatically added for each form, see notes above._ |
+| **Numeric** | Used for creating a numeric value field. |
+| **Blank Line** | Add a blank line separator to the page. |
+| **Page Break** | Used to start a new page in the instrument. |
+
+Click the *Add Row* button once all field information has been completed to submit the field to your instrument. A preview for every submitted field is displayed at the top and gets updated with every new addition. you can *Edit* or *Delete* fields from the preview panel if needed.
+
+#### Name and Download the instrument
+
+When you are ready to finalize your instrument or want to take a break from building, make sure to **save** your progress by clicking on the Save tab, provide a short name (Filename) for your instrument, and the full title of the instrument (Instrument Name).
+  
+> _**Note:** The Filename (aka `$TESTNAME`) should not contain hyphens, apostrophes, spaces, accents or special characters._
+
+Finally, click the `Save` button to download the `$TESTNAME.linst` file to your computer.
+
+## Validation in LINST
+
+Validation rules in the LINST format are implemented in a separate file from the instrument itself. The file containing the rules must be named exactly the same as the instrument however should have an `.rules` as opposed to the `.linst` extension. At the moment, there is no LORIS user interface available for the creation and editing of rule files, these must be created manually. A `rules` file uses the XIN formatting to parse and enforce the coded rules. Here is an example of the `bmi.rules` file drawn from the raisinbread demonstration codebase. 
```

Comment:
Is there documentation on XIN rules to link to? It's an internal LORIS standard so just saying "it uses XIN formatting" doesn't mean much to someone who doesn't know LORIS.